src/include/conf.h

```diff
@@ -12,10 +12,14 @@
  * Sample usage:
  *  __wt_conf_gets(session, conf, statistics, &cval);
  *  __wt_conf_gets(session, conf, Operation_tracking.enabled, &cval);
+ *  __wt_conf_getones(session, conf, cache_cursors, &cval));
  *  __wt_conf_gets_def(session, conf, no_timestamp, 0, &cval));
  */
 #define __wt_conf_gets(s, conf, key, cval) \
-    __wt_conf_gets_func(s, conf, WT_CONF_ID_STRUCTURE.key, 0, false, cval)
+    __wt_conf_gets_func(s, conf, WT_CONF_ID_STRUCTURE.key, 0, false, false, cval)
+
+#define __wt_conf_getones(s, conf, key, cval) \
+    __wt_conf_gets_func(s, conf, WT_CONF_ID_STRUCTURE.key, 0, false, true, cval)
```

Comment:
Can we add a `__wt_conf_getones_def` to match `__wt_config_getones_def`?  This is useful for anyone doing conversion in the future.  The implementation can be the same as `__wt_conf_gets_def`.  Again, since it isn't exactly equivalent, a comment would be helpful (similar to previous):
```
This is equivalent to the use of __wt_config_getones_def when using configuration strings for the case when there is a single configuration string supplied by the API caller, and a base "default" configuration string.  See previous comment for how more than two strings makes them different.
```
I think we're going to live with two config systems for a long time, but we'll probably be slowly doing more conversions.

Reply:
I'm a bit confused as we don't actually have a `__wt_config_getones_def`.